framework/include/vectorpostprocessors/LineMaterialSamplerBase.h

```diff
@@ -120,6 +120,8 @@ LineMaterialSamplerBase<T>::LineMaterialSamplerBase(const InputParameters & para
     _qrule(_subproblem.assembly(_tid).qRule()),
     _q_point(_subproblem.assembly(_tid).qPoints())
 {
+  _mesh.errorIfDistributedMesh("LineMaterialSamplerBase");
```

Comment:
is there an issue for this?

the code looks designed to work for distributed meshes

Reply:
I don't know why that line was added - @joshuahansel ?

Reply:
I don't know if there is an issue for it. I had just noticed that in the regression tests for it, it was restricted to replicated mesh.

Reply:
samplers in general are not restricted from distributed meshes. We need an issue if this line makes it in. But better if this line doesnt make it in if it s actually supported!

Reply:
Yep, that was my mistake - I should have made an issue for it. I'll make one now. I verified that something is definitely broken with it - just did not look into what it actually was.

Reply:
Here's the new issue: https://github.com/idaholab/moose/issues/21865. I'll reference it in comments in the code and `tests` files.

Reply:
Ok, I did this:

> I'll reference it in comments in the code and tests files.

So this conversation should now be resolved.